TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -746,13 +746,68 @@ public override bool Visit(DataDefinition dataDefinition)
                         "Group item " + dataDefinition.Name + " cannot have \"Blank when zero\" clause",
                         dataDefinitionEntry);
                 }
+            }
+            else if (commonDataDataDefinitionCodeElement != null) 
+            {
+                //This is an elementary item. It must have a picture, type or a valid usage
+                var levelNumberValue = levelNumber?.Value;
+                if ((levelNumberValue < 50 || levelNumberValue == 77)
+                    && dataDefinition.Picture == null
+                    //commonDataDataDefinitionCodeElement.UserDefinedDataType is the type as written in the code. dataDefinition.TypeDefinition is the resolved type which can be null if the type cannot be found
+                    && commonDataDataDefinitionCodeElement.UserDefinedDataType == null 
+                    && (!dataDefinition.Usage.HasValue || !IsUsageAllowedWithoutPicture(dataDefinition.Usage)))
```

Comment:
`!IsUsageAllowedWithoutPicture(dataDefinition.Usage.Value)` and change the signature of the local function to accept non-nullable `DataUsage`